lib/features/manual/widget/showcase_with_widget.dart

```diff
@@ -22,6 +22,7 @@ class ShowcaseWithWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final brightness = Theme.of(context).brightness;
```

Comment:
no need for this check, if you set the color `context.colorScheme.surface` then you don't need to set the font color at all, you can just leave it as default from the theme (by not setting it)